index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+    <title>Youtube</title>
+</head>
+<body>
+
+    <div class="container">
+        <div class="row">
+          <div class="col-sm-8">
+                <div class="embed-responsive embed-responsive-21by9">
+                    <iframe width="737" height="415" src="https://www.youtube.com/embed/mGNUaZekwMs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+              </div>
+              <h1 class="video-caption mt-3">Jake Paul vs. Anderson Silva second Presser: "I followed you back on Instagram! Follow me back!</h1>
+        </div>
+          <div class="col-sm-4">
+            <div class="row mt-3">
+                <div class="col-sm-6">
+                    <a href="https://www.youtube.com/watch?v=TNwnyA_N9ZM&t=1264s">
+                    <img src="images/image-2.png" class="img-fluid" alt="Responsive image">
+                    </a>
+              </div>
+                <div class="col-sm-6 ">
+                   <a href="https://www.youtube.com/watch?v=TNwnyA_N9ZM&t=1264s">
+                  <h3 class="text-list">Taiwan's SECRET WEAPON against China - A History of....</h3>
+                  <p class="text-list">John Coogan <br> 845K views 845K views</p>
+                   </a>
```

Comment:
Please use list elements such as unordered list 'ul' in this section as it makes the code more semantic.